☂️ v2

gregoralbrecht opened the pull request:
Two breaking changes:

1. A new implementation of `Tooltip` using Radix UI
2. Dropping IE11 support in layout components in favor of using flexbox's gap property

gregoralbrecht committed:
Use Radix UI for Tooltip

Our tooltip was previously using Reach UI, which still works perfectly
fine. However, the API of the Radix one is just ✨ so let's use it.

gsuess commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/qualifyze/design-system/237)
<!-- Reviewable:end -->

gregoralbrecht committed:
Drop IE11 for `Inline` in favor of flexbox's `gap`

We decided it's time to drop IE11 support for our `Inline` component in
favor of using `flexbox`'s `gap` property. Previously, we tried to avoid
`gap` (because it's not supported by IE11) but this has lead to some
gnarly issues with negative margins.

gregoralbrecht committed:
Drop IE11 for `Column` in favor of flexbox's `gap`

We decided it's time to drop IE11 support for our `Columns` component in
favor of using `flexbox`'s `gap` property. Previously, we tried to avoid
`gap` (because it's not supported by IE11) but this has lead to some
gnarly issues with negative margins.

gregoralbrecht committed:
Drop IE11 for `Tiles` in favor of flexbox's `gap`

We decided it's time to drop IE11 support for our `Tiles` component in
favor of using `flexbox`'s `gap` property. Previously, we tried to avoid
`gap` (because it's not supported by IE11) but this has lead to some
gnarly issues with negative margins.

gregoralbrecht committed:
Drop IE11 for `Stack` in favor of flexbox's `gap`

We decided it's time to drop IE11 support for our `Stack` component in
favor of using `flexbox`'s `gap` property. Previously, we tried to avoid
`gap` (because it's not supported by IE11) but this has lead to some
gnarly issues with negative margins.

gregoralbrecht committed:
Add changeset and release notes

gregoralbrecht committed:
Merge pull request #293 from Qualifyze/use-flexbox-gap-drop-ie11-support

gregoralbrecht reviewed (commented):
Verified that both changes work with this CSB: https://codesandbox.io/s/qf-design-system-forked-2ee2pu?file=/src/App.js

> Reviewed 24 of 24 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/qualifyze/design-system/237#-:-N0QqZrP-SwEiT6N5BhC:btfefbt)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @gregoralbrecht)



<!-- Sent from Reviewable.io -->
